src/alpaca_eval/models_configs/baize-v2-7b/configs.yaml

```diff
@@ -0,0 +1,13 @@
+baize-v2-7b:
+  prompt_template: "baize-v2-7b/prompt.txt"
```

Comment:
instead of duplicating can you use "baize-v2-13b/prompt.txt" here and remove the duplicated prompt?
Thanks